packages/kbn-search-api-panels/components/ingest_data.tsx

```diff
@@ -44,115 +43,42 @@ export const IngestData: React.FC<IngestDataProps> = ({
   sharePlugin,
   languages,
   consoleRequest,
+  http,
 }) => {
-  const [selectedIngestMethod, setSelectedIngestMethod] = useState<
-    'ingestViaApi' | 'ingestViaIntegration'
-  >('ingestViaApi');
+  const connectorPath = http.basePath.prepend('/app/connectors');
```

Comment:
suggestion: instead of creating the path here, add the path as an (optional?) input for the consuming component. These components are shared components and can be and are used both in Serverless and Dedicated--and this path doesn't exist in Dedicated deployments. So the Serverless and Enterprise Search plugins should each provide their own path to link to.

Incidentally, you will need to change the code in the Enterprise Search plugin too, or leave the old component in place. Enterprise Search will now break because it's importing a deleted component.

We should also consider whether we want these changes to apply to Dedicated at all (but that's a question for design, @julianrosado?).

Reply:
Based on discussion with @julianrosado ,  confirmed that `set up a connector` button should be redirected to [new connector page](https://www.figma.com/file/aRW3wtDRGgwOunC8vXIvdm/Milestone-0-UX?type=design&node-id=3961%3A491928&mode=design&t=mkvLCIam5eRkjVU6-1) . 

Since the [ingestData](https://github.com/elastic/kibana/blob/51da5af83162d9250be4abcbf53cdfb35a40a81f/packages/kbn-search-api-panels/components/ingest_data.tsx) component is shared among Dedicated & Serverless,  added a optional react component `additionalIngestionPanel ` to add any additional ingestion options. 

Thus, by default [ingestionsPanel](https://github.com/elastic/kibana/blob/51da5af83162d9250be4abcbf53cdfb35a40a81f/packages/kbn-search-api-panels/components/ingestions_panel.tsx) would have beats & log stash.  

Changes are added in this [commit](https://github.com/elastic/kibana/pull/171035/commits/e4614ad6b408a01e9115530385f2e29cde3e3c1e)

Reply:
Hmmm, odd, I swear the IntegrationsPanel was used in enterprise search when I last checked. I think the latest changes make sense and look good, though.